### PR TITLE
SimpleSAMLphp

### PR DESCRIPTION
**Pull request by jbickar**
@sherakama This should have no effect on existing Sites installs; simply adding simpleSAMLphp as an option (preparing for moving to cloud).

Please do not delete the simplesamlphp branch after merge.

### CODE REVIEW

**Comment by jbickar** [comment]
@mistermarco FYI

File: stanford.profile [hunk 1]
```diff
@@ -107,8 +107,18 @@ function stanford_sites_tasks() {
 
   //Enable the stanford_sites_helper module and the webauth module
   //Do this now rather than in .info file because it's looking for the administrator role and errors out otherwise
-  module_enable(array('stanford_sites_helper', 'webauth'));
-  
+  module_enable(array('stanford_sites_helper'));
```
Comment (sherakama) [inline stanford.profile:110]: We should wrap this in a check to see if on the sites environment.

File: stanford.profile [hunk 2]
```diff
@@ -107,8 +107,18 @@ function stanford_sites_tasks() {
...
+
+  // Enable our chosen authentication scheme
```
Comment (sherakama) [inline stanford.profile]: Comments should end on full stop.

File: stanford.profile (hunk 1 above)
Comment (sherakama) [inline stanford.profile:110]: And by we, I mean you.

Comment (jbickar) [inline stanford.profile:110]: Good thing we left the function that checks for that in here.

Comment (jbickar) [inline stanford.profile:110]: And by "we", I mean "I".

Comment (jbickar) [inline stanford.profile:110]: Hrm. Not sure if I agree on this, actually. Most of the functionality of stanford_sites_helper is useful outside of sites. If anything, we should wrap the AFS quota check in the stanford_sites_hosted() check. 

And by "we", I mean our future selves.

Comment (sherakama) [inline stanford.profile:110]: I agree with this. I have created a new branch for this work. Over time I wan't to move all of the environment specific code out of these profiles and into their own. We don't have a place for that right now so we will move this work to a 2.x branch and slow re-factor it all out.

**Comment by sherakama** [comment]
Moved to 2.x branch. Thanks.

**Pull request closed without merging**